services/app-api/handlers/reports/create.ts

```diff
@@ -77,13 +77,16 @@ export const createReport = handler(async (event, _context) => {
 
   const julyMcparRelease = unvalidatedMetadata?.julyMcparRelease;
 
+  const topicXIIUpdate = unvalidatedMetadata?.topicXIIUpdate;
+
   // eslint-disable-next-line no-useless-catch
   try {
     ({ formTemplate, formTemplateVersion } = await getOrCreateFormTemplate(
       reportBucket,
       reportType,
       isProgramPCCM,
-      julyMcparRelease
+      julyMcparRelease,
```

Comment:
`getOrCreateFormTemplate` has `julyMcparRelease` and `topicXIIUpdate` typed to `boolean` and required, so... (see next comment)